src/index.html

```diff
@@ -10,23 +10,41 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Airbnb</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="form"
+      action="#"
+      data-qa="big"
+    >
+      <div class="searchbar__container">
```

Comment:
Remember about correct indentation between parent and child elements. The `div` element should be indented two spaces inside the `form` element.